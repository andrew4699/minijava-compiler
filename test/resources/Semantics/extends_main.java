class Foo {
  public static void main(String[] args) {
    System.out.println(5 + 4 * 3 - 1);
  }
}

class A extends Foo {
  public int foo() {
    return 5;
  }
}

class B extends A {
  public int bar(Baz f, Bar b) {
    System.out.println(new A().method());
    return 5;
  }
}
