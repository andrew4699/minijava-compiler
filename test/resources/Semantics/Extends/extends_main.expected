[ExtendsVisitor - 7]: A extends an unknown class Foo
